examples/test.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-env mocha */
```

Comment:
There still ought to be a way to run the code, shouldn't there? Otherwise it's not a usable example. See title of the PR.